theme/scripts.html

```diff
@@ -1,48 +1,90 @@
 <script language="javascript" type="text/javascript">
```

Comment:
Remove `language` and `type` attributes. [`type` is optional](https://developer.mozilla.org/en/docs/Web/HTML/Element/script#attr-type) and `language` was already deprecated in HTML4.